pkg/apis/policy/validation/validation_test.go

```diff
@@ -807,6 +807,10 @@ func TestIsValidSysctlPattern(t *testing.T) {
 		"abc*",
 		"a.abc*",
 		"a.b.*",
+		"a/b/c/d",
+		"a/*",
+		"a/b/*",
+		"a.b/c*",
```

Comment:
Can we include these two patterns as well?
`
		"a.b/c.d",
		"a/b.c/d",
`

Reply:
Yes, we can include these two patterns as well.
I'll add them to the test cases.

Reply:
ping @mrunalp